app/components/Exchange/OrderBook.jsx

```diff
@@ -275,15 +282,20 @@ class GroupedOrderBookRowHorizontal extends React.Component {
                   order.amountToReceive().getAmount({real: true}),
                   base.get("precision")
               );
-        let total = isBid
+        const totalValueBids = quoteTotal ?
```

Comment:
Code duplication (see line 157). Can you refactor please?

Reply:
Are you sure @sschiessl-bcp ?

There are both `GroupedOrderBookRowHorizontal` and `OrderBookRowHorizontal`.